README.md

```diff
@@ -812,7 +812,18 @@ default in Linux can be anywhere from 20-120 seconds][linux-timeout]).
 
 ---
 
-- `time` - If `true`, the request-response cycle (including all redirects) is timed at millisecond resolution, and the result provided on the response's `elapsedTime` property. The `responseStartTime` property is also available to indicate the timestamp when the response begins.
+- `time` - If `true`, the request-response cycle (including all redirects) is timed at millisecond resolution, and the result provided on the response's `elapsedTime` property. The `responseStartTime` property is also available to indicate the timestamp when the response begins. In addition, there is a `.timings` object available with the following properties:
+  - `start`: Timestamp when `request()` was initialized
```

Comment:
+1 glad you thought about the deprecated behavior.

Can you add a comment near wherever `elapsedTime` and `responseStartTime ` are set in the code to indicate that they are now deprecated in favor of the `.timings` property? Once this has had some time to bake we can remove any README references to the old properties and think about how we deprecate them.